eden/scm/edenscm/ext/githelp.py

```diff
@@ -384,8 +384,10 @@ def clean(ui, repo, *args, **kwargs) -> None:
     args, opts = parseoptions(ui, cmdoptions, args)
 
     cmd = Command("clean")
+    if opts.get("d") or len(args) > 0:
+        cmd["--dirs"] = None
```

Comment:
For Sapling, `--dirs` only delete empty directories, so I think need to set both `--dirs` and `--files` option.